Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -294,7 +294,7 @@ def self.formula_ignores(formula)
 
     # Ignore matches to go keg, because all go binaries are statically linked.
     any_go_deps = formula.deps.any? do |dep|
-      dep.name =~ Version.formula_optionally_versioned_regex(:go)
+      dep.name&.match?(Version.formula_optionally_versioned_regex(:go))
```

Comment:
I'd rather a type signature disallowing nil in dependency.rb. There's probably bigger issues if this is ever nil.

Reply:
Thanks! We can just switch the order here, and it won't matter if dep.name is nil-able.